src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -640,4 +642,21 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return SettingsNav().Background();
     }
 
+    // If the theme asks for Mica, then drop out our background, so that we
+    // can have mica too.
+    void MainPage::_updateBackgroundForMica()

```

Comment:
```suggestion
    void MainPage::_UpdateBackgroundForMica()
```
nit: all the other ones start capitalized